cmd/test.go

```diff
@@ -0,0 +1,96 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/exercism/cli/workspace"
+	"github.com/spf13/cobra"
+)
+
+var testCmd = &cobra.Command{
+	Use:     "test",
+	Aliases: []string{"t"},
+	Short:   "Infer and run the test command for an exercise.",
+	Long: `Infer and run the test command for an exercise.
+
+	Run this command in an exercise's root directory.`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return runTest(args)
+	},
+}
+
+func runTest(args []string) error {
+	track, err := getTrack()
+
+	if err != nil {
+		return err
+	}
+
+	testConf, ok := workspace.TestConfigurations[track]
+
+	if !ok {
+		return fmt.Errorf("test handler for the `%s` track not yet implemented. Please see HELP.md for testing instructions", track)
```

Comment:
I also noticed that every track's testing instructions are at `https://exercism.org/docs/tracks/{TRACK}/tests`, so I could link out to that too (or instead).

Reply:
You could, but I think `HELP.md` is even better in this case

Reply:
yeah, great call. I went with `the \"%s\" track does not yet support running tests using the Exercism CLI. Please see HELP.md for testing instructions`, to be clear tests can be run via _a_ CLI, just not the exercism one. Also I kept the track name in, since that reminds them where they are (to reduce confusion vs "this track")